### ISSUE TITLE
build errors

### ISSUE THREAD

**Issue by caligolaz** [PredictionIO/template-scala-parallel-textclassification#5]
Hi there
 after 'pio build --verbose' a i get 10 compiling errors. I'm trying to compile after the data import following the tutorial here https://docs.prediction.io/demo/textclassification/

Error log

[INFO] [Console$] [info] Compiling 10 Scala sources and 1 Java source to /home/ubuntu/PredictionIO/bin/textclassification/target/scala-2.10/classes...
[INFO] [Console$] [error] /home/ubuntu/PredictionIO/bin/textclassification/src/main/scala/org/template/textclassification/BIDMachLRAlgorithm.scala:1: '{' expected but identifier found.
[INFO] [Console$] [error] package parallel-textclassification
[INFO] [Console$] [error]                 ^
[INFO] [Console$] [error] /home/ubuntu/PredictionIO/bin/textclassification/src/main/scala/org/template/textclassification/BIDMachLRAlgorithm.scala:178: '}' expected but eof found.
[INFO] [Console$] [error] /home/ubuntu/PredictionIO/bin/textclassification/src/main/scala/org/template/textclassification/DataSource.scala:1: '{' expected but identifier found.
[INFO] [Console$] [error] package parallel-textclassification
[INFO] [Console$] [error]                 ^
[INFO] [Console$] [error] /home/ubuntu/PredictionIO/bin/textclassification/src/main/scala/org/template/textclassification/DataSource.scala:144: '}' expected but eof found.

......

Best

**Comment by EmergentOrder** [comment]
See this Google Group thread: https://groups.google.com/forum/#!topic/predictionio-user/QIlYDKVB1pk.

tl;dr : package names cannot contain hyphens
